### DISCUSSION TITLE
npm install beta release

### DISCUSSION THREAD

**Discussion by Dreamsorcerer** [parcel-bundler/parcel#4848] [General] [answered]
Could we get a beta v2 release for npm?

I'm not really familiar with npm or parcel, so I've got no idea how to manually install the beta release.

I'm just trying to build an esmodule of the Open Layers project, which looks like it can only be done with parcel2.

**Comment by DeMoorJasper** [comment, marked as answer]
We've already published a v2 beta to npm, you can install it using `npm install parcel@next`

**Reply by Dreamsorcerer** [reply]
Oh, excellent. Thanks.

I was trying to install parcel-bundler, that must be why I couldn't find it.

I'll give it a go with that.

**Reply by DeMoorJasper** [reply]
Yeah, we no longer use the parcel-bundler name for v2
